### ISSUE TITLE
filter button

### ISSUE THREAD

**Issue by hoganmaps** [hoganmaps/treemap#14]
zac: -Tree Counts - I like the idea of having a small arrow to indicate there is more than just tree counts. I would (if this were something I was requesting) to have the tree counts in the upper right corner. Just personal preference, but the counts seem kind of hidden when tucked in the lower left.

**Comment by mcawley** [comment]
I agree - since you are looking up there for the zoom controls as well.

**Comment by hoganmaps** [comment]
Going to put a button to lock the filter/legend open
